src/components/LinkCard/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Flex, View, Link } from '@aws-amplify/ui-react';
+import { ExternalLinkIcon } from '../Icons';
+import styles from './link-card.module.css';
+
+export interface CardData {
+  name: string;
+}
+
+export interface LinkCardProps {
+  isExternal?: boolean;
+  href: string;
+  imageUrl: string;
```

Comment:
```suggestion
  imageUrl?: string;
```
I guess let's make this optional for now. We don't really know what the image will be and maybe it will be useful to be able to use LinkCard without a custom image.